### PR TITLE
Pipeline perms fix

### PR DESCRIPTION
**Pull request by glogiotatidis**
- unit tests pass, nevermind the frontend tests. they fail due to other changes.
- had to patch django-pipeline to respect Storages permissions. See also https://github.com/jazzband/django-pipeline/pull/570
- Had to remove virtualenv and re-create it to force update the packages

@jgmize r?

### CODE REVIEW

**Comment by safwanrahman** [comment]
We had broke the virtualenv and recreate it before and it made a lot of errors in the cronjob. So I would propose to not do it!

**Comment by jgmize** [comment]
@glogiotatidis I agree with both @safwanrahman's comment here and your comments in irc that virtualenv recreation should be done manually on an as-needed basis rather than on every deploy. Other than that and the failing tests this lgtm, and as you mentioned those are due to a separate issue, and while I'd of course prefer to see those passing first I'll leave it to your judgement on whether that should be a blocker for this. r+wc

File: requirements/default.txt [hunk 1]
```diff
@@ -31,19 +31,63 @@ celery==3.1.17
...
+# sha256: Kdkqca2Igv-fpUyXURLPEWqi5Zi9U90ji9chw-g2quA
+# sha256: VNOHaFdBZQnOtHkdk0fL32Gz_ZQPRr2ELP9A7VGE6QI
+# sha256: chp9gSb4ufTMJoviSAEC9zoCQM651tWcDCeT0KKE4CY
+# sha256: 2XkAjo_ptE0vyhEsxtNx5HeEAGSdVxDOaH0UaJ2qEII
+# sha256: hGQ4FZee5FqGdqoA5_dQEe3rM-VZvd0l5g6-p4uhLgo
+# sha256: I_7i45dhm8K8xIrf1rBF6Z7QxhiPqGCd01ajWgRgwZA
+# sha256: 0oL3ZesgmSbvEgjGUyYk-lCHpZjkCPincq6KZSXymHM
+# sha256: 9SRb08DUoxtK_Dl9FY7RsFG3A9lpBSdHLoyjcbINhzU
+# sha256: uxSVQO2QxLIXG_aU_mmR1jMbwUmuYjyP9BkyT0Ii0Sg
+cryptography==1.4
```
Comment (safwanrahman) [inline requirements/default.txt]: We have tried to upgrade the library before and it broke our developer site. Hope to concern about it.

**Comment by glogiotatidis** [comment]
I merged 
https://github.com/mozilla/kitsune/commit/25c4c1c9814882a2e9e8083dade8bbc6800fa3ff
and https://github.com/mozilla/kitsune/commit/6ae4ae24aa7814dc9af54aa711373f9feb8dfe2c and pushed to production.

We made only manual changes to the virtualenv to remove django-pipeline and to force install of the new version.

Things appear to be working.

Thanks for the feedback!

**Pull request closed without merging**